### PR TITLE
Adds unity+UWP installation instructions

### PR DESCRIPTION
**Pull request by sdolenc**
Overview
---
- Improves "getting started" documentation by 1) fixing existing section numbering, 2) adding "install unity's uwp integration" step, and 3) some other small edits

Changes
---
- Fixes: small documentation change. Probably not worth opening an issue

Additional Comments
---
- I don't know if htk_development is the best home for this. I couldn't find other protected development branches that had "GettingStarted.md". FWIW master also has GettingStarted.md
- The diff "rich view" preview may show the new image as 404'ing, but that's because I use relative pathing. It appears properly in [my fork](https://github.com/sdolenc/MixedRealityToolkit-Unity/blob/htk_development/GettingStarted.md) and will appear properly in the upstream Microsoft repo if the PR is merged

### CODE REVIEW

File: GettingStarted.md [hunk 1]
```diff
@@ -32,31 +35,33 @@ If you have not already, download and install [Visual Studio 2017](https://www.v
 
 You can install more components and UWP SDK's as you wish.
 
-Make sure you are running the appropriate version of Unity 3D on your machine. You should [download and install the latest version](https://unity3d.com/get-unity/download/archive) this project says it supports on the [main readme page](/README.md).
+## 2.3 Install appropriate version of Unity 3D
 
 [unity-release]:             https://unity3d.com/unity/qa/patch-releases/2017.2.1p2
```
Comment (davidkline-ms) [inline GettingStarted.md]: These should change to 2017.4 LTS

File: GettingStarted.md [hunk 2]
```diff
@@ -32,31 +35,33 @@ If you have not already, download and install [Visual Studio 2017](https://www.v
...
 ![Unity Installer](/External/ReadMeImages/UnityInstaller.PNG "Unity Installer")
 
-# 2. Download the MixedRealityToolkit-Unity asset packages
-You can download the latest unity package from [Releases](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases) folder.
+# 3. Download the MixedRealityToolkit-Unity asset packages
+
+### Option1: Download the unity package
 
-_Note: The latest release should work for both HoloLens and Windows Mixed Reality development._
+The latest [Release](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases) should work for both HoloLens and Windows Mixed Reality development.
```
Comment (davidkline-ms) [inline GettingStarted.md]: Can we make this HoloToolkit Release? Now that Mixed Reality Toolkit (aka vNext) is getting closer to release we should differentiate.

Thanks!

File: GettingStarted.md [hunk 3]
```diff
@@ -32,31 +35,33 @@ If you have not already, download and install [Visual Studio 2017](https://www.v
...
 [unity-release1]:                https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/latest
 [mrtk-version-badge]:            https://img.shields.io/github/tag/microsoft/MixedRealityToolkit-unity.svg?style=flat-square&label=Latest%20Master%20Branch%20Release&colorB=007ec6
 [![Github Release][mrtk-version-badge]][unity-release1]
 
-### Using the source code
-Optionally, If you'd like to build the Mixed Reality Toolkit from the source, you'll need to clone the GitHub repository from:
+### Option2: Use the source code
 
->  ## https://github.com/Microsoft/MixedRealityToolkit-Unity.git. 
+If you'd like to build the Mixed Reality Toolkit from the source, you'll need to clone the GitHub repository from:
```
Comment (davidkline-ms) [inline GettingStarted.md]: Let's use HoloToolkit (throughout) instead (see earlier comment)

File: GettingStarted.md [hunk 4]
```diff
@@ -68,10 +73,22 @@ If you've never cloned a repo before then you should consider using the GitHub d
...
 
+## 4.2 Install Unity's Universal Windows Platform integration
+
+`File -> Build Settings` select `Universal Windows Platform`
+
+If `Open Download Page` appears then save exe and install. Otherwise, dismiss modal window.
+
+![Unity Build Settings for UWP](/External/ReadMeImages/BuildSettingsUWP.png)
+
+## 4.3 Import MRTK asset
```
Comment (davidkline-ms) [inline GettingStarted.md]: Unity's menu is Import Package, let's use that term for clarity

File: GettingStarted.md [hunk 5]
```diff
@@ -128,7 +147,7 @@ Alternatively, you can setup your scene manually as follows:
 > **For Hololens**
 > Optionally, if you wish to enable spatial mapping in your scene for HoloLens, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
 
-# 5 Building your project 
+# 7. Building your project 
 The MRTK provides you a quick and easy way to generate your Unity project from a custom window, which can be found under:
```
Comment (davidkline-ms) [inline GettingStarted.md]: Let's replace MRTK with HTK (throughout) in this branch. Thanks!

Comment (sdolenc) [reply GettingStarted.md]: I fixed it for this file. Let me know if I should fix it globally for this branch

File: GettingStarted.md (hunk 4 above)
Comment (sdolenc) [reply GettingStarted.md]: fixed

File: GettingStarted.md (hunk 3 above)
Comment (sdolenc) [reply GettingStarted.md]: I switched it whenever the github project is reference, but didn't change it when it "Mixed Reality Toolkit" is a string displayed in the Unity UI.

File: GettingStarted.md (hunk 2 above)
Comment (sdolenc) [reply GettingStarted.md]: fixed

File: GettingStarted.md (hunk 1 above)
Comment (sdolenc) [reply GettingStarted.md]: Fixed for this file only. I suggest we move the whole github project forward to 2017.4 with a separate PR (tied to a github issue). That would be a good time to enforce version consistency with a simple CI test (ensuring version matches between markdown documentations and ProjectVersion.txt) .

**Pull request merged**